docs/events/gateway-events.mdx

```diff
@@ -1133,23 +1133,26 @@ Active sessions are indicated with an "online", "idle", or "dnd" string per plat
-| flags?          | integer                                                                              | [Activity flags](/docs/events/gateway-events#activity-object-activity-flags) `OR`d together, describes what the payload includes |
-| buttons?        | array of [buttons](/docs/events/gateway-events#activity-object-activity-buttons)     | Custom buttons shown in the Rich Presence (max 2)                                                                                |
+| Field                | Type                                                                                 | Description                                                                                                                                          |
+|----------------------|--------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| name                 | string                                                                               | Activity's name                                                                                                                                      |
+| type                 | integer                                                                              | [Activity type](/docs/events/gateway-events#activity-object-activity-types)                                                                          |
+| url?                 | ?string                                                                              | Stream URL, is validated when type is 1                                                                                                              |
+| created_at           | integer                                                                              | Unix timestamp (in milliseconds) of when the activity was added to the user's session                                                                |
+| timestamps?          | [timestamps](/docs/events/gateway-events#activity-object-activity-timestamps) object | Unix timestamps for start and/or end of the game                                                                                                     |
+| application_id?      | snowflake                                                                            | Application ID for the game                                                                                                                          |
+| status_display_type? | ?integer                                                                             | [Status display type](/docs/events/gateway-events#activity-object-status-display-types); controls which field is displayed in the user's status text |
```

Comment:
we might want to specify that this applies to the member list - "status text" is a bit ambiguous